src/html/boldOrItalic.js

```diff
@@ -1,10 +1,129 @@
 'use strict';
 
 var strings = require('../strings');
-var wrapping = require('./wrapping');
 
 function boldOrItalic (chunks, type) {
-  wrapping(type === 'bold' ? 'strong' : 'em', strings.placeholders[type], chunks);
+  wrapping_bold_italic(type === 'bold' ? 'strong' : 'em', strings.placeholders[type], chunks);
```

Comment:
Will it be best to stick to the naming style on the codebase? camelCase in this case.

Reply:
ok